bee-network/bee-gossip/src/swarm/behaviour.rs

```diff
@@ -43,13 +43,13 @@ impl NetworkBehaviourEventProcess<IdentifyEvent> for SwarmBehaviour {
     fn inject_event(&mut self, event: IdentifyEvent) {
         match event {
             IdentifyEvent::Received { peer_id, info } => {
-                trace!(
-                    "Received Identify request from {}. Observed address: {:?}.",
-                    alias!(peer_id),
-                    info.observed_addr,
-                );
+                trace!("Received Identify response from {}: {:?}.", alias!(peer_id), info,);
 
-                // TODO: log supported protocols by the peer (info.protocols)
+                // Panic:
+                // Sending must not fail.
```

Comment:
For panic comments I would usually expect the reason why we can be sure it will not fail.
Just stating it must not fail is redundant with the fact we have an expect in the first place.

Reply:
Also, I would put both comments in the same line.

Reply:
you're right :+1: